.evergreen/config.yml

```diff
@@ -300,6 +299,15 @@ functions:
           - .evergreen/scripts/run-with-env.sh
           - .evergreen/scripts/run-tests.sh
 
+  "run direct tests":
+    - command: subprocess.exec
+      type: test
+      params:
+        working_dir: "src"
+        binary: bash
+        include_expansions_in_env: ["PYTHON_BINARY"]
+        args: [ .evergreen/scripts/run-direct-tests.sh ]
```

Comment:
Why do we need a new entry point to test free threading?

Reply:
Hatch doesn't work with it, because it depends on cryptography.  It made more sense to me to add this specific entry point then try to hack back in a "SKIP_HATCH" flag.

Reply:
Thanks SGTM